.env.example

```
@@ -0,0 +1,173 @@
+# Levels: NONE, ERROR, WARN, INFO, DEBUG, TRACE
+LOGGING_LEVEL=INFO
+
+# ===== System Monitoring =====
+# System monitoring thresholds (1.0 = disabled, default: 0.8)
+# Set < 1.0 to enable monitoring (e.g., 0.8 for 80% threshold)
+MAX_CPU=0.8
+MAX_RAM=0.8
+
+# ===== Performance Configuration =====
+# Vector search performance settings
+MAX_THRESHOLD_HISTORY=10
```

dotenv-linter (3.3.0)

[warning] [DuplicatedKey] The MAX_THRESHOLD_HISTORY key is duplicated

(DuplicatedKey)